packages/support/src/Commands/InstallCommand.php

```diff
@@ -37,7 +37,7 @@ public function __invoke(): int
 
         if (confirm(
             label: 'All done! Would you like to show some love by starring the Filament repo on GitHub?',
-            default: true,
+            default: ! $this->option('no-interaction'),
```

Comment:
Instead, maybe we should early return before this even happens, so we don't need to ask the question in the first place

Reply:
I've extracted a method in which it returns early if the `no-interaction` option is passed. I've tested it and it has the same desired effect.